lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -127,10 +125,9 @@ def allocate_delay
     end
 
     def allocate_delay=(val)
-      @set_params = {}
-      @set_params[:state] = val ? '' : 'no'
-      @set_params[:delay] = val ? val : ''
-      config_set('fabricpath', 'allocate_delay', @set_params)
+      state = val ? '' : 'no'
+      delay = val ? val : ''
+      config_set('fabricpath', 'allocate_delay', state: state, delay: delay)
     rescue Cisco::CliError => e
```

Comment:
Is it only for this specific case? I see it for other existing code in node-utils and it is a cut-n-paste from those. I can remove from this one.